### PR TITLE
[RW-7318][risk=low] Implement admin egress events API

### PR DESCRIPTION
**Pull request by calbach**

### CODE REVIEW

**Comment by calbach** [comment]
@yonghaoy ptal

File: api/src/main/java/org/pmiops/workbench/api/EgressEventsAdminController.java [hunk 1]
```diff
@@ -1,20 +1,105 @@
...
+              .orElseThrow(
+                  () ->
+                      new NotFoundException(
+                          String.format(
+                              "workspace namespace '%s' not found",
+                              request.getSourceWorkspaceNamespace())));
+    }
+
+    Page<DbEgressEvent> page;
+    if (userFilter == null && workspaceFilter == null) {
```
Comment (yonghaoy) [inline api/src/main/java/org/pmiops/workbench/api/EgressEventsAdminController.java:77]: nit: It's bit 'backward' while reading the code. Can it be:
```
if(workspaceFilter != null && userFilter != null) {
 findAllByUserAndWorkspaceOrderByCreationTimeDesc
} 
else if(workspaceFilter != null) {
  findAllByWorkspace
} else if(userFilter != null) {
 findByUser
} else {
 findAll? 
```
}

**Review by yonghaoy — APPROVED** [review]
LG! Sorry to missed your Friday afternoon PR request again...

File: api/src/main/java/org/pmiops/workbench/api/EgressEventsAdminController.java (hunk 1 above)
Comment (calbach) [reply api/src/main/java/org/pmiops/workbench/api/EgressEventsAdminController.java]: Good call, much cleaner

**Pull request merged**